TESTING.rst

```diff
@@ -152,46 +152,192 @@ You can also specify individual tests or a group of tests:
 
     ./breeze --test-target tests/test_core.py::TestCore
 
-Running Full test suite via scripts from the host
--------------------------------------------------
 
-To run all tests with default settings (Python 3.6, Sqlite backend, "docker" environment), enter:
+Airflow Integration Tests
+=========================
 
-.. code-block::
+Some of the tests in Airflow are Integration tests. Those tests require not only airflow-testing docker
+image but also extra images with integrations (such as redis/mongodb etc.).
 
-  ./scripts/ci/local_ci_run_airflow_testing.sh
 
+Enabling integrations
+---------------------
 
-To select Python 3.6 version, Postgres backend, and a ``docker`` environment, specify:
+Running Airflow integration tests cannot be run in local virtualenv. They can only run in Breeze
+environment with enabled integrations and in Travis CI.
+
+When you are in Breeze environment, by default all integrations are disabled - this way only true unit tests
+can be executed in Breeze. You can enable the integration by passing ``--integration <INTEGRATION>``
+switch when starting Breeze. You can specify multiple integrations by repeating the ``--integration`` switch
+or by using ``--integration all`` switch which enables all integrations.
+
+Note, that every integration requires separate container with the corresponding integration image,
+so they take precious resources on your PC - mainly memory. The integrations started are not stopped
+until you stop the Breeze environment with ``--stop-environment`` switch.
+
+The following integrations are available:
+
+.. list-table:: Airflow Test Integrations
+   :widths: 15 80
+   :header-rows: 1
+
+   * - Integration
+     - Description
+   * - cassandra
+     - Integration required for Cassandra hooks
+   * - kerberos
+     - Integration that provides Kerberos authentication
+   * - mongo
+     - Integration required for MongoDB hooks
+   * - openldap
```

Comment:
Note to reviewers: I could not find any openldap integration tests. Maybe some of them are simply skipped now. I'd love if you can point me to the tests that should be marked with pytest.mark.integration("openldap")